src/examples/java/com/devcycle/examples/LocalExample.java

```diff
@@ -46,5 +52,6 @@ public static void main(String[] args) throws InterruptedException {
         } else {
             System.out.println("feature is NOT enabled");
         }
+        Thread.sleep(10000);
```

Comment:
why this long sleep?

Reply:
To actually connect to SSE and pull a config